### PR TITLE
[#1526] Show course entry requirements

### PR DESCRIPTION
**Pull request by dankmitchell**
### Context
Course minimum GSCE requirements

### Changes proposed in this pull request
Render "Course minimum GSCE requirements" on course basic page 

### Guidance to review
Example - `/organisations/2CG/courses/2YXH`

### CODE REVIEW

File: app/views/courses/_basic_details_tab.html.erb [hunk 1]
```diff
@@ -114,6 +114,90 @@
...
       </div>
 
+      <div class="govuk-summary-list__row">
+        <dt class="govuk-summary-list__key">
+          UCAS applications: Maths GCSE
+        </dt>
+        <dd class="govuk-summary-list__value">
+          <% case course.maths %>
+          <% when 'must_have_qualification_at_application_time' %>
+            <p class="govuk-body">1 - UCAS will block applications from candidates who:</p>
```
Comment (fofr) [inline app/views/courses/_basic_details_tab.html.erb]: Can we tweak this slightly for each code:
```html
<p>Code: 1</p>
<p>UCAS will block...</p>
```

File: app/views/courses/_basic_details_tab.html.erb [hunk 2]
```diff
@@ -114,6 +114,90 @@
...
+            <ul class="govuk-list govuk-list--bullet">
+              <li>haven't gained the minimum qualifications yet</li>
+              <li>need to take an equivalency test</li>
+            </ul>
+          <% when 'expect_to_achieve_before_training_begins' %>
+            <p class="govuk-body">2 - Recieve applications from candidates who haven't gained minimum qualifications yet.</p>
```
Comment (fofr) [inline app/views/courses/_basic_details_tab.html.erb]: Typo: Recieve > Receive

Comment (fofr) [inline app/views/courses/_basic_details_tab.html.erb]: Typo: Recieve > Receive

File: app/views/courses/_basic_details_tab.html.erb [hunk 3]
```diff
@@ -114,6 +114,90 @@
...
+            <p class="govuk-body">UCAS will block applications from candidates who take an equivalency test.</p>
```
Comment (fofr) [inline app/views/courses/_basic_details_tab.html.erb]:
```suggestion
            <p class="govuk-body">UCAS will block applications from candidates who need to take an equivalency test.</p>
```

File: app/views/courses/_basic_details_tab.html.erb [hunk 4]
```diff
@@ -114,6 +114,90 @@
...
+      <div class="govuk-summary-list__row">
+        <dt class="govuk-summary-list__key">
+          UCAS applications: Maths GCSE
+        </dt>
+        <dd class="govuk-summary-list__value">
+          <% case course.maths %>
+          <% when 'must_have_qualification_at_application_time' %>
+            <p class="govuk-body">1 - UCAS will block applications from candidates who:</p>
+            <ul class="govuk-list govuk-list--bullet">
+              <li>haven't gained the minimum qualifications yet</li>
```
Comment (fofr) [inline app/views/courses/_basic_details_tab.html.erb]:
```suggestion
              <li>haven’t gained the minimum qualifications yet</li>
```

File: app/views/courses/_basic_details_tab.html.erb (hunk 2 above)
Comment (fofr) [inline app/views/courses/_basic_details_tab.html.erb]:
```suggestion
            <p class="govuk-body">2 - Recieve applications from candidates who haven't gained the minimum qualifications yet.</p>
```

File: app/views/courses/_basic_details_tab.html.erb [hunk 5]
```diff
@@ -114,6 +114,90 @@
...
+              <li>need to take an equivalency test</li>
+            </ul>
+          <% when 'expect_to_achieve_before_training_begins' %>
+            <p class="govuk-body">2 - Recieve applications from candidates who haven't gained minimum qualifications yet.</p>
+            <p class="govuk-body">UCAS will block applications from candidates who take an equivalency test.</p>
+          <% when 'equivalence_test' %>
+            <p>3 - Recieve applications from candidates who:</p>
```
Comment (fofr) [inline app/views/courses/_basic_details_tab.html.erb]:
```suggestion
            <p>3 - Receive applications from candidates who:</p>
```

File: app/views/courses/_basic_details_tab.html.erb [hunk 6]
```diff
@@ -114,6 +114,90 @@
...
+            <ul class="govuk-list govuk-list--bullet">
+              <li>haven't gained the minimum qualifications yet</li>
```
Comment (fofr) [inline app/views/courses/_basic_details_tab.html.erb]:
```suggestion
              <li>haven’t gained the minimum qualifications yet</li>
```

File: app/views/courses/_basic_details_tab.html.erb [hunk 7]
```diff
@@ -114,6 +114,90 @@
...
+              <li>need to take an equivalency test</li>
+            </ul>
+            <p class="govuk-body">Reject candidates who need to take an equivalency test</p>
```
Comment (fofr) [inline app/views/courses/_basic_details_tab.html.erb]: This line needs to be deleted
```suggestion

```

**Comment by dankmitchell** [comment]
[Basic details: show maths/english/science settings](https://trello.com/c/YL5ESIfm/1526-basic-details-show-maths-english-science-settings)

File: app/views/courses/_entry_requirements.html.erb [hunk 8]
```diff
@@ -0,0 +1,23 @@
+<% case subject %>
```
[the remark this answers is not in the document]
Comment (dankmitchell) [reply app/views/courses/_entry_requirements.html.erb]: I think `gcse_subject` is probably best here

File: app/views/courses/_entry_requirements.html.erb [hunk 9]
```diff
@@ -0,0 +1,23 @@
+<% case gcse_subject %>
+<% when 'must_have_qualification_at_application_time' %>
+  <p class="govuk-body">Code: 1</p>
+  <p class="govuk-body">UCAS will block applications from candidates who:</p>
+  <ul class="govuk-list govuk-list--bullet">
+    <li>haven't gained the minimum qualifications yet</li>
```
Comment (fofr) [inline app/views/courses/_entry_requirements.html.erb]:
```suggestion
    <li>haven’t gained the minimum qualifications yet</li>
```

File: app/views/courses/_entry_requirements.html.erb [hunk 10]
```diff
@@ -0,0 +1,23 @@
...
+    <li>need to take an equivalency test</li>
+  </ul>
+<% when 'expect_to_achieve_before_training_begins' %>
+  <p class="govuk-body">Code: 2</p>
+  <p class="govuk-body">Receive applications from candidates who haven't gained minimum qualifications yet.</p>
```
Comment (fofr) [inline app/views/courses/_entry_requirements.html.erb]:
```suggestion
  <p class="govuk-body">Receive applications from candidates who haven’t gained the minimum qualifications yet.</p>
```

File: app/views/courses/_entry_requirements.html.erb [hunk 11]
```diff
@@ -0,0 +1,23 @@
...
+  <p class="govuk-body">UCAS will block applications from candidates who take an equivalency test.</p>
+<% when 'equivalence_test' %>
+  <p class="govuk-body">Code: 3</p>
+  <p class="govuk-body">Receive applications from candidates who:</p>
+  <ul class="govuk-list govuk-list--bullet">
+    <li>haven't gained the minimum qualifications yet</li>
```
Comment (fofr) [inline app/views/courses/_entry_requirements.html.erb]:
```suggestion
    <li>haven’t gained the minimum qualifications yet</li>
```

File: app/views/courses/_entry_requirements.html.erb [hunk 12]
```diff
@@ -0,0 +1,23 @@
+<% case gcse_subject %>
```
Comment (fofr) [inline app/views/courses/_entry_requirements.html.erb]: The value of this is a code, not a subject name. Maybe `gcse_subject_code`?

**Pull request merged**